modules/logging-release-notes-5-9-7.adoc

```diff
@@ -0,0 +1,30 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-7_{context}"]
+= Logging 5.9.7
+This release includes link:https://access.redhat.com/errata/RHBA-2024:7324[OpenShift Logging Bug Fix Release 5.9.7].
+
+
+[id="openshift-logging-5-9-7-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter was not applied to the Fluentd configuration when Fluentd was used as the collector type, causing HTTP timeouts to be misconfigured. With this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter is now correctly applied, ensuring Fluentd honors the specified timeout and handles HTTP connections according to the user’s configuration. (link:https://issues.redhat.com/browse/LOG-6125[LOG-6125])
+
+* Before this update, the TLS section was added without verifying the broker URL schema, resulting in SSL connection errors if the URLs did not start with `tls`. With this update, the TLS section is now added only if the broker URLs start with `tls`, preventing SSL connection errors. (link:https://issues.redhat.com/browse/LOG-6041[LOG-6041])
```

Comment:
It would be much better here to avoid the passive voice and write that something (whatever it is) added/adds the TLS section. Ideally also where the URLs come from. This would give much more context as to what is actually happening.